dm/dm/config/task_cli_args_test.go

```diff
@@ -43,3 +43,12 @@ func (t *testConfig) TestTaskCliArgsDowngrade(c *C) {
 	c.Assert(afterDowngrade.Decode([]byte(data)), IsNil)
 	c.Assert(afterDowngrade.StartTime, Equals, "123")
 }
+
+func (t *testConfig) TestTaskCliArgsVerify(c *C) {
+	empty := TaskCliArgs{}
+	c.Assert(empty.Verify(), IsNil)
+	wrongStartTime := TaskCliArgs{StartTime: "2006-01-02T15:04:05"}
+	c.Assert(wrongStartTime.Verify(), NotNil)
+	rightStartTime := TaskCliArgs{StartTime: "2006-01-02 15:04:05"}
+	c.Assert(rightStartTime.Verify(), IsNil)
```

Comment:
Why not support both of them, seems it's easy to implement.

Reply:
cc @sunzhaoyang

Reply:
And 2006-01-02T15:04:05 is better then 2006-01-02 15:04:05 because it has not quote and space.

Reply:
+1